tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/identifier/IdentifierLiteralTest.java

```diff
@@ -56,17 +58,22 @@ public void testModuleIdentifierClash() throws BallerinaTestException {
         runLeecher.waitForText(5000);
     }
 
-    @Test(description = "Test clashes in organization and module names that contain '_'", enabled = false)
+    @Test(description = "Test clashes in organization and module names that contain '_'")
     public void testPackageIDClash() throws BallerinaTestException {
-        Path projectPath = Paths.get(testFileLocation, "PackageNameClashProject")
-                .toAbsolutePath();
-        Path importProjectPath = Paths.get(testFileLocation, "testProject")
-                .toAbsolutePath();
-        LogLeecher runLeecher = new LogLeecher("2 passing");
-        bMainInstance.runMain("build", new String[0], new HashMap<>(), new String[0],
-                new LogLeecher[]{}, importProjectPath.toString());
-        bMainInstance.runMain("build", new String[0], new HashMap<>(), new String[0],
-                new LogLeecher[]{runLeecher}, projectPath.toString());
+        String importProjectPath = testFileLocation + File.separator + "testProject";
+        LogLeecher buildLeecher = new LogLeecher("target/bala/a_b-foo-any-0.1.0.bala");
+        LogLeecher pushLeecher =
+                new LogLeecher("Successfully pushed target/bala/a_b-foo-any-0.1.0.bala to 'local' repository.", ERROR);
+        LogLeecher runLeecher = new LogLeecher("Tests passed");
+        bMainInstance.runMain("build", new String[]{"-c"}, null, null, new LogLeecher[]{buildLeecher},
```

Comment:
Previously we tested with ballerina test command and now have been changes to run command.

Reply:
Shall we use the same approch?